WalletWasabi.Fluent/ViewModels/Wallets/Home/History/HistoryItems/HistoryItemViewModelBase.cs

```diff
@@ -87,6 +87,9 @@ protected string GetConfirmedToolTip(int confirmations)
 
 	public bool IsCPFPd { get; set; }
 
+	public bool IsIncomingTransactionDisplayed => IsConfirmed && (IncomingAmount != null && IncomingAmount > Money.Zero);
+	public bool IsOutgoingTransactionDisplayed => IsConfirmed && (OutgoingAmount != null && OutgoingAmount > Money.Zero);
+	public bool IsSelfShareTransaction => TransactionSummary.Transaction.IsSelfTransfer;
```

Comment:
I would rename this to `IsSelfTransferTransaction`

Reply:
Hello @yahiheb , thank you, I have renamed it.